src/charts/histogramBoxPlot/onResize.js

```diff
@@ -99,9 +155,9 @@ export default function onResize() {
           })
```

Comment:
I'm thinking the box plot might be a little more elegant without these vertical lines.  Let me know what you think.